### PR TITLE
Prepare the repository

### PR DESCRIPTION
**Pull request by lucasmichot**
This PR keeps `actions/checkout` on `v3`
But make sure to prepare the repository

cf. https://github.com/SocialiteProviders/Providers/issues/820

### CODE REVIEW

**Review by atymic — APPROVED** [review]

**Comment by atymic** [comment]
Actually think it might have been this:
https://github.com/SocialiteProviders/Providers/commit/c5fbdf233a9e35abe2189457df79d6391938fd9e#diff-f3de2374c2be9ebdce1e259e31e4bbe5129f8c7682ab7e87ef41b774d0b14465L16-L18

**Comment by atymic** [comment]
Reverting it to v1 seemed to fix the issue. Guess we need to look at changelogs and figure out how to get it working on new vers.

File: .github/workflows/monorepo-split.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
```
Comment (atymic) [inline .github/workflows/monorepo-split.yml:14]: I tried with v3 and it didn't work sadly, had to roll all the way back to v1 to get the correct commits 🤔 
https://github.com/SocialiteProviders/Providers/commit/e95fb7091ed1aa2ef526b4e70d157673f192b3c5

Comment (lucasmichot) [reply .github/workflows/monorepo-split.yml:14]: 👍

**Pull request closed without merging**